analyze/analyses/dominators/emit.rs

```diff
@@ -0,0 +1,297 @@
+use std::collections::BTreeMap;
```

Comment:
I moved the `Emit` implementation out into a separate submodule in the same manner as the paths analysis. That file was becoming a little hefty, and the emit logic was upstaging the analysis itself. That's in a separate commit to make reviewing easier :)